hooks/post-command

```diff
@@ -27,6 +28,10 @@ if [ "${BUILDKITE_PLUGIN_CODECOV_FAIL_JOB_ON_ERROR:-${default_fail_job_on_error}
     codecov_args+=(--nonZero)
 fi
 
+if [ "${BUILDKITE_PLUGIN_CODECOV_FLAGS:-${default_flags}}" != "true" ]; then
+    codecov_args+=(--flags="${BUILDKITE_PLUGIN_CODECOV_FLAGS:-}")
+fi
```

Comment:
It looks like the default flag value "true" isn't actually used anywhere, except as a sentinel value meaning "nothing was set for BUILDKITE_PLUGIN_CODECOV_FLAGS". 

This is different from the above `BUILDKITE_PLUGIN_CODECOV_FAIL_JOB_ON_ERROR` where that default _is_ consumed by the interior of the if statement.

As such: I think you'd be better off removing `default_flags` and changing this line to
```
if [[ -v BUILDKITE_PLUGIN_CODECOV_FLAGS ]] ; then
```
where -v is a "does this variable exist?" check. 

(last note: unexpectedly, you don't want any $ or {} on the -v check, just the identifier of the variable)

Reply:
Makes sense, updated.